client/modules/accounts/templates/profile/profile.js

```diff
@@ -2,14 +2,19 @@ import * as Collections from "/lib/collections";
 import { Meteor } from "meteor/meteor";
 import { Template } from "meteor/templating";
 
+let allShops = [];
```

Comment:
We actually make it a global variable, and it is not a const, it changes. Thanks for the feeback